ui/app/src/components/ItemStateIcon/ItemStateIcon.tsx

```diff
@@ -106,6 +109,10 @@ const useStyles = makeStyles(() =>
     stateDisabledIcon: (styles) => ({
       color: palette.pink.main,
       ...styles.stateDisabledIcon
+    }),
+    stateStandby: (styles) => ({
```

Comment:
What's "standby"? Is this for the not in workflow state? Use a semantic name, not the name of the icon you're using.